ui/admin/app/styles/app.scss

```diff
@@ -1054,3 +1054,14 @@
   overflow: hidden;
   text-overflow: ellipsis;
 }
+
+#remove-tag-modal {
```

Comment:
not opposed to this - but I don't think we use IDs for styles in our repo, curious why ID over classnames

Reply:
Absolutely right. Updated.